components/dropdown/Dropdown.tsx

```diff
@@ -115,7 +121,10 @@ const Dropdown: React.FC<DropdownProps> = ({
                     ? styles.active
                     : styles.listOfCities
                 }
-                onClick={handleClick}
+                onClick={(e) => {
+                  e.preventDefault();
+                  chooseSuggestion(suggestion);
```

Comment:
The type of this functions is now just `(suggestion: string) =>`  (a little easier to grasp I think)